tuist/tuist
# docs: fix #target-scripts anchor in Manifests.Project

Resolves https://github.com/tuist/tuist/issues/4020

### Short description 📝

Fixes [#target-scripts](https://docs.tuist.io/manifests/project/#target-scripts) anchor in documentation.

### How to test the changes locally 🧐

- cd to [docs](./projects/docs) folder
- run `npm run build && npm run serve`
- open [Manifests->Projects](http://localhost:3000/manifests/project) documentation
- Find the url with `[TargetScript]` text and make sure it redirects you to the proper anchor

### Checklist ✅
